### ISSUE TITLE
Can't remove entries from file allowed_extensions list

### ISSUE THREAD

**Issue by robingram** [silverstripe/silverstripe-assets#438]
Using Silverstripe in 4.6.

The documented method here to remove existing entries from the `allowed_extensions` list does not work: https://docs.silverstripe.org/en/4/developer_guides/files/allowed_file_types/

If I add, say:

```
SilverStripe\Assets\File:
  allowed_extensions:
    zip: false
```

Looking at the code, all it does is skip that new entry for `zip`. It doesn't check if zip is already in the list and remove it.

If that is the intended behaviour then perhaps the docs need to be clarified.

**Comment by maxime-rainville** [comment]
Fixed by https://github.com/silverstripe/silverstripe-assets/pull/439

**Comment by jonners99** [comment]
Is this fix going to be in a tagged release soon?

**Comment by maxime-rainville** [comment]
@jonners99  It's going to be included in the next 4.8 release which should ship early June-ish. If you need it right now, I can create a 1.7.x release of `assets`
